client/src/api/api.js

```diff
@@ -32,10 +33,45 @@ const signInCall = async (user) => {
   }
 };
 
+const createFriendList = async (friendList) => {
+  try {
+    const result = await fetch('http://localhost:3001/api/friendLists/lists', {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify(friendList),
+      credentials: 'include',
+    });
+    // console.log(JSON.stringify(friendList));
+    console.log('result', result);
+    const response = await result.json();
+    console.log('response', response);
+    return response;
+  } catch (err) {
+    return err;
+  }
+};
+
+const editFriendList = async (friendList) => {
+  try {
+    const result = await fetch('http://localhost:3001/api/friendLists/lists', {
+      method: 'PUT',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify(friendList),
+      credentials: 'include',
```

Comment:
In the backend for the PUT request, the list id is being passed as part of the URL params, not the request body
```suggestion
     const listId = friendList.id;
    const result = await fetch(`http://localhost:3001/api/friendLists/lists/${listId}`, {
      method: 'PUT',
      headers: { 'Content-Type': 'application/json' },
      body: JSON.stringify(friendList),
      credentials: 'include',
```